docs/source/installation/index.rst

```diff
@@ -0,0 +1,112 @@
+.. _installation:
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+
+   guide_quick/index
+   guide_complete/index
+   docker
+   troubleshooting
+
+============
+Installation
+============
+
+Before you can use AiiDA, you will need to get it installed:
+
+.. grid:: 1 2 2 2
+   :gutter: 3
+
+   .. grid-item-card:: :fa:`rocket;mr-1` Quick install
+      :text-align: center
+      :shadow: md
+
+      Install AiiDA in the most simple way that should work on most systems.
+      Choose this method if you are new to AiiDA and simply want to try it out.
+
+      +++++++++++++++++++++++++++++++++++++++++++++
+
+      .. button-ref:: guide_quick/index
+         :ref-type: doc
+         :click-parent:
+         :expand:
+         :color: primary
+         :outline:
+
+         To the quick installation guide
+
+   .. grid-item-card:: :fa:`info-circle;mr-1` Complete installation guide
+      :text-align: center
+      :shadow: md
+
+      Install AiiDA with full control over the configuration.
+      Choose this method if you are an advanced user or you want to optimize the setup for your system.
+
+      +++++++++++++++++++++++++++++++++++++++++++++
+
+      .. button-ref:: guide_complete/index
+         :ref-type: doc
+         :click-parent:
+         :expand:
+         :color: primary
+         :outline:
+
+         To the complete installation guide
+
+
+.. admonition:: Problems during installation?
+    :class: warning
+
+    If you encountered any issues, please refer to the :ref:`troubleshooting section <installation:troubleshooting>`.
+
+
+.. admonition:: Installation successful
+    :class: hint
+
+    If you are a new user, we recommend to start with the :ref:`basic tutorial <tutorial:basic>`.
+    Alternatively, check out the :ref:`next steps guide <tutorial:next-steps>`.
+
+
+Preinstalled environments
```

Comment:
I would for now not include it in `aiida-core` to keep dev and release cycle simpler by having it be decoupled. What else is needed for a deployment? I would like to test it to see how it works. We played around with it a long time ago and it was essentially not worth much. It would take a huge amount of time on startup and maintenance costs were not negligible. But happy to have a look now if things have changed